.teamcity/common/Os.kt

```diff
@@ -43,4 +43,8 @@ enum class Os(val agentRequirement: String, val ignoredSubprojects: List<String>
     );
 
     fun escapeKeyValuePair(key: String, value: String) = if (this == WINDOWS) """$key="$value"""" else """"$key=$value""""
+
+    fun lowerCase() = name.toLowerCase()
```

Comment:
I think these methods shouldn't say what they do but what purpose the string the produce have:

I think we need some version of the `Os.name` for
- Displaying it in the UI (-> `toString()`)
- Use as part of Teamcity IDs (-> `asId` or something like that, depending on what we use in other places)
- As part of the Teamcity property for the Java home. Maybe that could go to another class/method or be something like `toJavaHomePropertyId()`.

WDYT?